core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -266,19 +300,19 @@ object StorageTool extends Logging {
   }
 
   private def getUserScramCredentialRecord(
-    mechanism: String,
-    config: String
-  ) : UserScramCredentialRecord = {
+                                            mechanism: String,
+                                            config: String
+                                          ) : UserScramCredentialRecord = {
     /*
      * Remove  '[' amd ']'
      * Split K->V pairs on ',' and no K or V should contain ','
      * Split K and V on '=' but V could contain '=' if inside ""
      * Create Map of K to V and replace all " in V
      */
     val argMap = config.substring(1, config.length - 1)
-                       .split(",")
-                       .map(_.split("=(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)"))
-                       .map(args => args(0) -> args(1).replaceAll("\"", "")).toMap
+      .split(",")
+      .map(_.split("=(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)"))
+      .map(args => args(0) -> args(1).replaceAll("\"", "")).toMap
```

Comment:
Please preserve the old indentation. In Kafka that is how we indent 2 space when the expression spans multiple lines.